### ISSUE TITLE
Add an option to enable replication crash safety without GTID

### ISSUE THREAD

**Issue by datacharmer** [datacharmer/dbdeployer#36]
Currently, to set replication crash safety, we use `--gtid`, which adds all the needed options (but See issue #35 for more.)

We want an option that enables crash safety without using GTID, and without spelling out all the options explicitly.

The new option, called `--repl-crash-safe` will include:
```
master-info-repository=table
relay-log-info-repository=table
relay-log-recovery=on
```

**Comment by jfg956** [comment]
As a small comment, "master-info-repository=table" is not needed for replication crash-safety.  I do not think it should be removed, just pointing-out it is not needed.  More about this in (1).

(1): https://jfg-mysql.blogspot.com/2016/08/discussion-about-sync-master-info-and-replication-parameters.html

**Comment by datacharmer** [comment]
Implemented in release 1.11.0

Thanks to @jfg956 for bringing the issue to my attention.
